app/views/hyrax/embargoes/_list_active_embargoes.html.erb

```diff
@@ -0,0 +1,16 @@
+<table class="embargoes table">
+  <tbody>
+  <tr>
+    <th>Type of Work</th><th>Title</th><th>Current Visibility</th><th>Embargo Release Date</th><th>Visibility will Change to</th>
```

Comment:
Then at the very least, I'd like to see that ticketed, although I think it'd be easier in the long run to at least extract these into an `en` i18n file, instead of having to dive through all our code later.